.gitignore

```diff
@@ -12,9 +15,14 @@ __pycache__
 .pytest_cache
 .python-version
 *.egg-info
+.venv/
 dist
+.venv
 
 ## Terraform ##
 .terraform
 .terraform.lock.hcl
 .tfvars
+
+## MacOS ##
+.DS_Store
```

Comment:
These duplicate lines can be deleted:
```suggestion
```

Reply:
@dav3r resolved with [63734a9](https://github.com/cisagov/windows-commando-packer/pull/10/commits/63734a9525c10a930199dca1f49346023cd2c5c7)

Reply:
👍  Friendly reminder that you can use the GitHub web GUI's "Commit suggestion" feature to quickly commit simple suggested changes like this one.  If you do use that feature, please be sure to replace the default commit message (e.g. `Update .gitignore`) with a more useful one (e.g. `Remove duplicate lines`).